ocaml/tests/test_client.ml

```diff
@@ -0,0 +1,42 @@
+
+(** This module tests more layers of xapi than other tests by using the Client
+    module to make XenAPI calls. It tests many of the auto-generated files as
+    these XenAPI calls go through the client, server.ml, message forwarding,
+    and database layers. *)
+
+let setup_test () =
+  let __context = Test_common.make_test_database () in
+  Test_common.make_client_params ~__context
+
+let test_create () =
+  let rpc, session_id = setup_test () in
+  let self = Client.Client.Task.create ~rpc ~session_id ~label:"task_label" ~description:"task_description" in
+  Alcotest.(check string) "task label"
```

Comment:
This string could be a bit more descriptive about where a possible assertion failure would be raised.

Reply:
Hmm OK, what should I write here?
However, for me the main point of these strings is to locate which assertion failed within a test case, but this test, `test_create`, has only one assertion, this is why I did not give much thought to to what string I should use here.

Reply:
That's true, it just looked a bit confusing at first because the string matches one of your assertion arguments. I think anything like "check task_label in test_create" works.

Reply:
Which unit test this assertion belongs to will be reported by Alcotest if it fails :)

Reply:
This is why Alcotest is awesome